eng/common/scripts/add-pullrequest-reviewers.ps1

```diff
@@ -18,6 +18,28 @@ param(
     $AuthToken
 )
 
+function AddMembers($memberName, $additionSet) {
+  $headers = @{
+    Authorization = "bearer $AuthToken"
+  }
+  $uri = "https://api.github.com/repos/$RepoOwner/$RepoName/pulls/$PRNumber/requested_reviewers"
+
+  foreach ($id in $additionSet) {
+    try {
+      $postResp = @{}
+      $postResp[$memberName] = @($id)
+      $postResp = $postResp | ConvertTo-Json
+
+      Write-Host $postResp
+      $resp = Invoke-RestMethod -Method Post -Headers $headers -Body $postResp -Uri $uri -MaximumRetryCount 3
+      $resp | Write-Verbose
+    }
+    catch {
+      Write-Error "Error attempting to add $user `n$_"
```

Comment:
By default DevOps will stop on the first error. If you want to attempt to write them all you might need to save your Write-Error for the end.

Reply:
I did not know that tidbit. I'll change them to Write-Host, along with a final bool for failure.